ocs_ci/ocs/ocp.py

```diff
@@ -185,6 +189,8 @@ def exec_oc_debug_cmd(self, node, cmd_list, timeout=300):
         )
         if err_msg in out:
             raise CommandFailed
+        elif "error:" in out:
```

Comment:
Is this sufficient to know that the debug pod was failed to create or is there a more explicit error message we can try to look for? Do you have an example of the `out` in this case?

Reply:
"error" is sufficient in this case. I saw below 2 use cases for debug pod failures
1.
```
18:02:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc debug nodes/ip-10-0-149-60.us-east-2.compute.internal -- df
18:02:42 - ThreadPoolExecutor-381_0 - ocs_ci.utility.utils - WARNING - Command stderr: Error from server (NotFound): pods "pod-test-cephfs-2e71ee0092804860aad497c93d6c7f5d" not found

18:03:34 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Starting pod/ip-10-0-149-60us-east-2computeinternal-debug ...
To use host binaries, run `chroot /host`

Removing debug pod ...
error: Back-off pulling image "registry.redhat.io/rhel7/support-tools"
```
and
2.
```
2022-01-07 09:07:07,112 - MainThread - INFO - ocs_ci.utility.utils.exec_cmd.487 - Executing command: oc debug nodes/compute-7 -- chroot /host /bin/bash -c "ifconfig $(route | grep default | awk '{print $(NF)}') down || echo 'CMD FAILED'; "
2022-01-07 09:07:07,698 - MainThread - DEBUG - ocs_ci.utility.utils.exec_cmd.502 - Command stdout is empty
2022-01-07 09:07:07,698 - MainThread - WARNING - ocs_ci.utility.utils.exec_cmd.506 - Command stderr: Starting pod/compute-7-debug ...
To use host binaries, run `chroot /host`

Removing debug pod ...
error: unable to create the debug pod "compute-7-debug"
```